libpromises/evalfunction.c

```diff
@@ -251,7 +264,8 @@ static JsonElement* VarRefValueToJson(EvalContext *ctx, const FnCall *fp, const
                     }
                     else if (var->ref->num_indices - index_offset > 1)
                     {
-                        Log(LOG_LEVEL_DEBUG, "%s: got ref with starting depth %zd and index count %zd", fp->name, index_offset, var->ref->num_indices);
+                        Log(LOG_LEVEL_DEBUG, "%s: got ref with starting depth %zd and index count %zd",
+                            fp ? fp->name : "VarRefValueToJson", index_offset, var->ref->num_indices);
```

Comment:
I made a variable `fp_name` because, the way we use `fp`, I think it may be needed in the future so just passing a string is not future-proof. I hope that's OK.